tests/api/test_search.py

```diff
@@ -87,26 +86,34 @@ def test_temporal_search_open():
     assert search.end_date is None
 
 
-def test_invalid_temporal_search():
-    # Not RFC339
-    utcnow = datetime.utcnow().strftime("%Y-%m-%d")
-    with pytest.raises(ValidationError):
-        Search(collections=["collection1"], datetime=utcnow)
+# def test_invalid_temporal_search_date():
+#     # Just a date, no time
+#     utcnow = datetime.now(timezone.utc).strftime("%Y-%m-%d")
+#     with pytest.raises(ValidationError):
+#         Search(collections=["collection1"], datetime=utcnow)
+
```

Comment:
This might have stay depending on the outcome of https://github.com/stac-utils/stac-pydantic/pull/131